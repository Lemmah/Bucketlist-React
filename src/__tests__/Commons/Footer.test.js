import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../Components/Commons/Footer';

it('Displays footer without crashing', () => {
  const rendered = renderer.create(
    <Footer />,
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});
