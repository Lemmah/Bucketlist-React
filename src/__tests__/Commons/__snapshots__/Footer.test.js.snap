// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays footer without crashing 1`] = `
<div>
  <div
    style={
      Object {
        "display": "block",
        "height": "60px",
        "padding": "20px",
        "width": "100%",
      }
    }
  />
  <div
    style={
      Object {
        "backgroundColor": "#F8F8F8",
        "borderTop": "1px solid #E7E7E7",
        "bottom": "0",
        "height": "60px",
        "left": "0",
        "padding": "20px",
        "position": "fixed",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    <p>
      Â© 2017 Lemayian - Dream Buckets. 
    </p>
  </div>
</div>
`;
